apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -111,7 +111,8 @@ const TutorialDetail = React.createClass({
     closeClicked: PropTypes.func.isRequired,
```

Comment:
Want to update to `extends React.Component` while you're in the file or save for follow up?

Reply:
Good question.  I'd like to do that in a standalone PR, just to isolate concerns.